private-templates-service/client/src/components/Common/AdaptiveCard/AdaptiveCard.tsx

```diff
@@ -115,7 +120,7 @@ class AdaptiveCard extends React.Component<Props> {
         }
       }
       if (template.length === 0) {
-        template = processTemplate(this.props.cardtemplate.instances[0]);
+        template = processTemplate(this.props.cardtemplate.instances[this.props.cardtemplate.instances.length - 1]);
```

Comment:
Is this meant to render the oldest version or the most recent version? Versions are ordered from most recent to least

Reply:
Oh didn't know that, will change it back